src/DurableTask.SqlServer/SqlOrchestrationQuery.cs

```diff
@@ -53,5 +53,10 @@ public class SqlOrchestrationQuery
         /// Gets or sets an instance ID prefix to use for filtering orchestration instances.
         /// </summary>
         public string? InstanceIdPrefix { get; set; }
+
+        /// <summary>
+        /// Determines whether the query will retrieve only parent instances.
+        /// </summary>
+        public bool FetchParentInstancesOnly { get; set; } = false;
```

Comment:
Naming: I suggest `FetchRootInstancesOnly`.

Reply:
+1, or maybe `ExcludeSubOrchestrations`, which I like because it's just a verb and a noun. I also think something like `ExcludeSubOrchestrations` more clearly explains what this setting is doing.

The main problem with `FetchParentInstancesOnly` is that sub-orchestrations can also be parents if they create sub-orchestrations of their own, so the current name isn't actually completely accurate based on my reading of the code.